tests/5.0/target_update/test_target_update_mapper_from_discontiguous.F90

```diff
@@ -0,0 +1,68 @@
+!===--- test_target_update_mapper_from_discontiguous.F90 ----------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+!  This test seeks to ensure that target update with motion-clause "to"
```

Comment:
Description does not match test, I will check C test as I think we may have bungled it there.